Octokit/Models/Request/SearchIssuesRequest.cs

```diff
@@ -337,4 +336,74 @@ public enum IssueTypeQualifier
         [Parameter(Value = "issue")]
         Issue
     }
+
+    public class RepositoryCollection : IEnumerable<string>
```

Comment:
Well there's List<T> that'll work. :stuck_out_tongue: Also, it looks like this is supposed to be a mutable collection right now, so maybe List<T> would make more sense than `ReadOnlyCollection<T>`. Or we could go the immutable route and have `Add` always return a new collection.

I just think having this implement `IEnumerable<T>` is not right. At minimum, `ICollection<T>` would make more sense.